storybook/stories/components/search/results.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+import React from 'react'
+import { Col, Grid } from 'reflex'
+import { times } from 'lodash'
+import styles from '../../../shared/stories.css'
+
+export default function SearchResults({
+  results = 1,
+  pages = 4,
+}: {
+  results: number,
+  pages: number,
+}) {
+  return (
+    <Grid size={9} className={styles.colors2} align="top">
```

Comment:
Same thing than with filters, this is specific to the search page, I think it would make sense to put here a single search result and have the search page use it.